Lidarr Update Error

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Lidarr

### 📂 What was the exact command used to execute the script?

update

### ⚙️ What settings are you using?

- [ ] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📝 Provide a clear and concise description of the issue.

Attempting to update in verbose mode using the update command gives the following error:

environment: line 32: jq: command not found
curl: (23) Failure writing output to destination

[ERROR] in line 32: exit code 0: while executing command RELEASE=$(curl -fsSL https://api.github.com/repos/Lidarr/Lidarr/releases/latest | jq -r '.tag_name' | sed 's/^v//')



### 🔄 Steps to reproduce the issue.

Running update command

### ❌ Paste the full error output (if available).

environment: line 32: jq: command not found
curl: (23) Failure writing output to destination

[ERROR] in line 32: exit code 0: while executing command RELEASE=$(curl -fsSL https://api.github.com/repos/Lidarr/Lidarr/releases/latest | jq -r '.tag_name' | sed 's/^v//')



### 🖼️ Additional context (optional).

Have previously updated with no issues.

--- a/ct/lidarr.sh
+++ b/ct/lidarr.sh
@@ -29,7 +29,7 @@ function update_script() {
     exit
   fi
 
-  RELEASE=$(curl -fsSL https://api.github.com/repos/Lidarr/Lidarr/releases/latest | jq -r '.tag_name' | sed 's/^v//')
+  RELEASE=$(curl -fsSL https://api.github.com/repos/Lidarr/Lidarr/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
   if [[ "${RELEASE}" != "$(cat ~/.lidarr)" ]] || [[ ! -f ~/.lidarr ]]; then
 
     msg_info "Stopping service"

--- a/ct/lidarr.sh
+++ b/ct/lidarr.sh
@@ -28,8 +28,12 @@ function update_script() {
     msg_error "No ${APP} Installation Found!"
     exit
   fi
+  if ! command -v jq &>/dev/null; then
+    $STD apt-get update
+    $STD apt-get install -y jq
+  fi
 
-  RELEASE=$(curl -fsSL https://api.github.com/repos/Lidarr/Lidarr/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+  RELEASE=$(curl -fsSL https://api.github.com/repos/Lidarr/Lidarr/releases/latest | jq -r '.tag_name' | sed 's/^v//')
   if [[ "${RELEASE}" != "$(cat ~/.lidarr)" ]] || [[ ! -f ~/.lidarr ]]; then
 
     msg_info "Stopping service"

